docs/pages/overview.md

```diff
@@ -147,13 +153,55 @@ app = rx.App()
 
```

Comment:
for the example above I think we should use add_page if this is the approach we are trying to push (again I think we should have one way to do it and push them in the docs, and then we can show an alternative, but it can get confusing if you are showing two ways to do it everywhere)